crates/bevy_sprite/src/bundle.rs

```diff
@@ -41,6 +43,10 @@ pub struct SpriteBundle {
 /// Check the following examples for usage:
 /// - [`animated sprite sheet example`](https://github.com/bevyengine/bevy/blob/latest/examples/2d/sprite_sheet.rs)
 /// - [`texture atlas example`](https://github.com/bevyengine/bevy/blob/latest/examples/2d/texture_atlas.rs)
```

Comment:
These links to the examples should be added to `SpriteBundle` documentation (and `ImageBundle` in its respective file too).

Reply:
(forgot that `ButtonBundle` also includes `UiImage`, so the example links would be duplicated between _three_ bundles and the component itself)

Reply:
Okay, I'll leave it out for now.